packages/flutter_tools/lib/src/commands/fuchsia_reload.dart

```diff
@@ -354,41 +356,26 @@ class FuchsiaReloadCommand extends FlutterCommand {
 }
 
 
-// TODO(zra): When Fuchsia has ssh, this should be changed to use that instead.
 class FuchsiaDeviceCommandRunner {
+  // TODO: Get rid of _address and instead use
+  // $_fuchsiaRoot/out/build-magenta/tools/netaddr --fuchsia
+  final String _address;
+  final String _buildType;
   final String _fuchsiaRoot;
-  final Random _rng = new Random(new DateTime.now().millisecondsSinceEpoch);
 
-  FuchsiaDeviceCommandRunner(this._fuchsiaRoot);
+  FuchsiaDeviceCommandRunner(this._address, this._fuchsiaRoot, this._buildType);
 
   Future<List<String>> run(String command) async {
-    final int tag = _rng.nextInt(999999);
-    const String kNetRunCommand = 'out/build-magenta/tools/netruncmd';
-    final String netruncmd = fs.path.join(_fuchsiaRoot, kNetRunCommand);
-    const String kNetCP = 'out/build-magenta/tools/netcp';
-    final String netcp = fs.path.join(_fuchsiaRoot, kNetCP);
-    final String remoteStdout = '/tmp/netruncmd.$tag';
-    final String localStdout = '${fs.systemTempDirectory.path}/netruncmd.$tag';
-    final String redirectedCommand = '$command > $remoteStdout';
-    // Run the command with output directed to a tmp file.
-    ProcessResult result =
-        await Process.run(netruncmd, <String>[':', redirectedCommand]);
-    if (result.exitCode != 0)
-      return null;
-    // Copy that file to the local filesystem.
-    result = await Process.run(netcp, <String>[':$remoteStdout', localStdout]);
-    // Try to delete the remote file. Don't care about the result;
-    Process.run(netruncmd, <String>[':', 'rm $remoteStdout']);
-    if (result.exitCode != 0)
+    final String config = '$_fuchsiaRoot/out/$_buildType/ssh-keys/ssh_config';
+    final List<String> args = <String>['-F', config, _address, command];
+    printTrace('ssh ${args.join(' ')}');
+    final ProcessResult result =
+        await Process.run('ssh', <String>['-F', config, _address, command]);
```

Comment:
Use `processManager.run()` so this can be mocked out in tests.

Then, can you add a test? :-)